### PR TITLE
Reformat Cross

### PR DESCRIPTION
**Pull request by Ajay6601**

### CODE REVIEW

File: ivy/functional/ivy/linear_algebra.py [hunk 1]
```diff
@@ -180,6 +180,14 @@ def cross(
...
+         
+    This method conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/extensions/generated/signatures.linalg.cross.html>`_ # noqa
+    in the standard. The descriptions above assume an array input for simplicity, but
+    the method also accepts :code:`ivy.Container` instances in place of
+    :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
+    and also the examples below.
 
     Functional Examples
```
Comment (VedPatwardhan) [inline ivy/functional/ivy/linear_algebra.py:192]: This section should be named `Examples`.

**Comment by Ajay6601** [comment]
Hi,
Can you able to tell me the requested changes need to be made?

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @Ajay6601, thanks for contributing, there are a few changes to be made to your PR before we get it merged along with the one I've suggested near the line of code concerned. :
`functional/ivy/linear_algebra.py`: 
1. The examples inside `functional/ivy/linear_algebra.py` should not need a header like on line 197.
2. Please remove the examples under `With ivy.NativeArray inputs:`
3. Please update the examples with `With one ivy.Container input` and `With multiple ivy.Container inputs`.
4. Please adjust the header of examples on line 224 to be on a single line.
5. Please remove the examples under `With a combination of ivy.NativeArray and ivy.Array inputs`.
6. Instance method examples shouldn't be in `functional/ivy/linear_algebra.py` please remove them from there.

Let me know if you have any questions. Thanks 🙂

**Comment by VedPatwardhan** [comment]
Also, please type in the number of the issue you've created so that I can link it to your PR, thanks 🙂

**Comment by Ajay6601** [comment]
#2138

**Comment by Ajay6601** [comment]
Still is there anything need to be changed.

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @Ajay6601, thanks for making the changes, they look fine to me. However, the docstring tests weren't working yesterday because of a segmentation fault because of which we won't be able to see the docstring test results for your PR. They are working now, so you can just merge the current master code into your branch and re-request my review so that we can take a look at the tests.
Let me know if you have any questions. Thanks 🙂

**Comment by Ajay6601** [comment]
I have merged with branch

File: ivy/array/linear_algebra.py [hunk 2]
```diff
@@ -75,6 +75,42 @@ def cross(
...
+        
+        Parameters
+        ----------
+        self
+            first input array. Should have a numeric data type.
+        x2
+            second input array. Must be compatible with ``self``
+            (see :ref:`broadcasting`). Should have a numeric data type.
+        axis
+            the axis (dimension) of x1 and x2 containing the vectors for which to compute
```
Comment (VedPatwardhan) [inline ivy/array/linear_algebra.py:91]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/array/linear_algebra.py [hunk 3]
```diff
@@ -75,6 +75,42 @@ def cross(
...
+            (default: -1) the cross product.vIf set to -1, the function computes the 
+            cross product for vectors defined by the last axis (dimension). Default: -1.
+        out
+            optional output array, for writing the result to. It must have a shape that
+            the inputs broadcast to.
+            
+        Returns
+        -------
+        ret
+            an array containing the element-wise products. The returned array must have a
```
Comment (VedPatwardhan) [inline ivy/array/linear_algebra.py:101]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/container/linear_algebra.py [hunk 4]
```diff
@@ -211,6 +211,69 @@ def static_cross(
...
+        
+        Parameters
+        ----------
+        x1
+            first input array. Should have a numeric data type.
+        x2
+            second input array. Must be compatible with ``self``
+            (see :ref:`broadcasting`). Should have a numeric data type.
+        axis
+            the axis (dimension) of x1 and x2 containing the vectors for which to compute
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:227]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/container/linear_algebra.py [hunk 5]
```diff
@@ -211,6 +211,69 @@ def static_cross(
...
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        out
+            optional output container, for writing the result to. It must have a shape
+            that the inputs broadcast to.
+            
+        Returns
+        -------
+        ret
+            an array containing the element-wise products. The returned array must have a
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:250]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/container/linear_algebra.py [hunk 6]
```diff
@@ -234,6 +297,56 @@ def cross(
...
+        
+        Parameters
+        ----------
+        x1
+            first input array. Should have a numeric data type.
+        x2
+            second input array. Must be compatible with ``self``
+            (see :ref:`broadcasting`). Should have a numeric data type.
+        axis
+            the axis (dimension) of x1 and x2 containing the vectors for which to compute
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:313]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/container/linear_algebra.py [hunk 7]
```diff
@@ -234,6 +297,56 @@ def cross(
...
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        out
+            optional output container, for writing the result to. It must have a shape
+            that the inputs broadcast to.
+            
+        Returns
+        -------
+        ret
+            an array containing the element-wise products. The returned array must have a
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:336]: This line is longer than 88 characters, which is why it fails the lint tests

File: ivy/container/linear_algebra.py [hunk 8]
```diff
@@ -234,6 +297,56 @@ def cross(
...
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.cross. 
+        This method simply wraps the function, and so the docstring 
+        for ivy.cross also applies to this method with minimal changes.
+        
+        Parameters
+        ----------
+        x1
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:307]: this argument should be `self` here.

File: ivy/container/linear_algebra.py [hunk 9]
```diff
@@ -234,6 +297,56 @@ def cross(
...
+            first input array. Should have a numeric data type.
+        x2
+            second input array. Must be compatible with ``self``
+            (see :ref:`broadcasting`). Should have a numeric data type.
+        axis
+            the axis (dimension) of x1 and x2 containing the vectors for which to compute 
+            (default: -1) the cross product.vIf set to -1, the function computes the 
+            cross product for vectors defined by the last axis (dimension). Default: -1.
+        out
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:316]: `out` is added twice.

File: ivy/container/linear_algebra.py [hunk 10]
```diff
@@ -211,6 +211,69 @@ def static_cross(
...
```
Comment (VedPatwardhan) [inline ivy/container/linear_algebra.py:230]: `out` is added twice.

File: ivy/functional/ivy/linear_algebra.py [hunk 11]
```diff
@@ -180,90 +180,50 @@ def cross(
...
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/extensions/generated/signatures.linalg.cross.html>`_ # noqa
+    in the standard. The descriptions above assume an array input for simplicity, but
+    the method also accepts :code:`ivy.Container` instances in place of
+    :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
+    and also the examples below.
 
-    Functional Examples
+    Examples
     --------
```
Comment (VedPatwardhan) [inline ivy/functional/ivy/linear_algebra.py:194]: there shouldn't be a blank line here

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @Ajay6601, thanks for making the changes. I have suggested some more changes doing which would get the lint and numpy style docstring tests passing apart from the ones that are already failing on master. The docstring tests are already passing for our function.
Let me know if you have any questions. Thanks 🙂

**Comment by Ajay6601** [comment]
Hi,
It seems like lint tests are passed.

**Review by VedPatwardhan — APPROVED** [review]
lgtm! Thanks for contributing 🙂

**Pull request merged**